index.Rmd

```diff
@@ -59,7 +59,7 @@ SWWdf<-occ2df(SWW)
 ```
 Spocc downloads data via "occ" and converts to a dataframe via "occ2df".
 
-# Results
+# Results 
```

Comment:
see my notes in the pull request - I would encourage you to not add a shiny app until you have your analysis worked out.  I suspect that will take most of your time.